apps/src/templates/ProtectedStatefulDiv.jsx

```diff
@@ -4,7 +4,8 @@ import PropTypes from 'prop-types';
 import Radium from 'radium'; // eslint-disable-line no-restricted-imports
 
 /**
- * A div DOM element that will never update its contents and will throw an
+ * A div DOM element that will never update its contents unless the non-standard
+ * DOM attribute, canupdate, is assigned 'yes' It will throw an
```

Comment:
nit: 'canUpdate' is no longer a "non-standard" DOM attribute right? mind updating this comment?

Reply:
Thanks for catching!